pkg/limayaml/defaults.go

```diff
@@ -551,9 +562,24 @@ func FillDefault(y, d, o *LimaYAML, filePath string) {
 		y.MountType = o.MountType
 	}
 	if y.MountType == nil || *y.MountType == "" {
-		if *y.VMType == VZ {
+		switch *y.VMType {
+		case VZ:
 			y.MountType = ptr.Of(VIRTIOFS)
-		} else {
+		case QEMU:
+			y.MountType = ptr.Of(NINEP)
+			// Use REVSSHFS if the instance was created with Lima prior to v1.0
+			if _, err := os.Stat(filePath); err == nil { // existing instance
+				switch {
+				case existingLimaVersion == "": // created with Lima < 0.20
+					y.MountType = ptr.Of(REVSSHFS)
```

Comment:
This case is identical to the default case, so you could just update the comment there.

Reply:
done